### PR TITLE
Add NavigationDrawer Component and link to SYNAPSS

### PR DESCRIPTION
**Pull request by KoukiMinamoto**
I created NavigationDrawer on mobile and profileRate Banner on profile page.
But, since profileRate banner is still WIP so I commented it out for now.

スクリーンショット 2023-07-05 13 59 52
スクリーンショット 2023-07-05 14 01 12

### CODE REVIEW

File: src/components/organism/Modal/Onboarding/PFOnboardingForm.tsx [hunk 1]
```diff
@@ -0,0 +1,384 @@
+import styled from '@emotion/styled'
```
Comment (kitakaze-kan) [inline src/components/organism/Modal/Onboarding/PFOnboardingForm.tsx:1]: Can you please stop committing these WIP changes? A large number of unrelated commits will make future reviews difficult...

File: package.json [hunk 2]
```diff
@@ -55,6 +56,7 @@
     "react-device-detect": "^2.2.2",
     "react-dom": "18.2.0",
     "react-hook-form": "^7.42.1",
+    "react-icons": "^4.9.0",
```
Comment (kitakaze-kan) [inline package.json:59]: where do you use it?

**Review by kitakaze-kan — CHANGES_REQUESTED** [review]

**Comment by KoukiMinamoto** [comment]
I recreated a pull request here.
#28

**Comment by kitakaze-kan** [comment]
can we delete this branch? @KoukiMinamoto

**Pull request closed without merging**